src/main/java/com/ge/predix/uaa/token/lib/FastTokenServices.java

```diff
@@ -180,10 +180,16 @@ private void verifyIssuer(final String iss) {
         }
     }
 
-    private void verifyTimeWindow(final Map<String, Object> claims) {
+    protected void verifyTimeWindow(final Map<String, Object> claims) {
```

Comment:
does this need to be 'protected'?

Reply:
@sanjeevchopra, to expose the method to the test.  I could have it package-protected instead, I guess...

Reply:
yes, please do